required/latex-lab/latex-lab-footnotes.dtx

```diff
@@ -1854,13 +1853,13 @@
          {
            \exp_not:N\fnote_gput_refs:ee
             { \tag_get:n{struct_num} }
-            { \ref_value:Vn\l_@@_currentlabel_tl {fnote/struct} }
+            { \property_ref:ee{ @@/\l_@@_currentlabel_tl } {fnote/struct} }
          }
 %    \end{macrocode}
 %    in this case we set the name of the linktarget in the note
 %    to the structure number of the text mark.
 %    \begin{macrocode}
-        \tl_set:Nx \l_@@_linktarget_tl {footnote*.\ref_value:Vn\l_@@_currentlabel_tl {fnote/struct}}
+        \tl_set:Nx \l_@@_linktarget_tl {footnote*.\property_ref:ee {@@/\l_@@_currentlabel_tl} {fnote/struct}}
```

Comment:
:Nx -> :Ne   and :ee -> :en ?

Reply:
I think we are holding off on `\tl_set:Nx`, etc., for a bit as we want to avoid `expl3` issues as far as possible for users

Reply:
@josephwright while that might be an issue with normal packages, I don't see the need for latex-lab code. That usually requires a current L3 prog layer anyway, I would have thought.